tools/topology/topology2/include/components/eqfir/passthrough.conf

```diff
@@ -0,0 +1,14 @@
+Object.Base.data."fir_eq" {
+	bytes "
+		0x53,0x4f,0x46,0x34,0x00,0x00,0x00,0x00,
+		0x38,0xe0,0x00,0x00,0x04,0x00,0x00,0x00,
```

Comment:
0x38, 0x00, 0x00, 0x00.
The 0x0e is wrong.

Reply:
urgh indeed. fixed now.